memory.js

```diff
@@ -0,0 +1,201 @@
+var Game = function(boardElements) {
+	this.boardElements = boardElements;
+	this.tileClass = boardElements.tileHtmlElementClass.substr(1);
+	this.maxValue = boardElements.height*boardElements.width/2;
+	this.takenValues = "#";
+	this.values = null;
+	this.clicked1 = null;
+	this.clicked2 = null;
+	this.remainingMatches = boardElements.height*boardElements.width/2;
+};
+
+Game.prototype.taken = function(value) {
```

Comment:
Suggestion: methods that return `true`/`false` are often prefixed with `is`, i.e. `isTaken()`.